src/main/charts/common/templates/_jmx.tpl

```diff
@@ -30,8 +30,15 @@ Jmx init container
   image: {{ .Values.monitoring.jmxExporterImageRepo}}:{{ .Values.monitoring.jmxExporterImageTag}}
   command: ["cp"]
   args: ["/opt/bitnami/jmx-exporter/jmx_prometheus_javaagent.jar", "{{ .Values.volumes.sharedHome.mountPath }}"]
+  {{- if .Values.monitoring.jmxExporterInitContainer.runAsRoot }}
```

Comment:
Maybe I am just blind but I can't see the `monitoring.jmxExporterInitContainer.runAsRoot` and `monitoring.jmxExporterInitContainer.customSecurityContext` stanza in any of the `values.yaml` files. Was this just omitted in the PR?

Reply:
I see, that does make sense. Might be just good to indicate in PR description.